resources/views/components/exhibition-hero.blade.php

```diff
@@ -0,0 +1,28 @@
+@if (!empty($hero))
+    <div class="parallax home-hero exhibition-hero">
+        @if(!empty($hero['image']))
+            <div class="bg-overlay" style="background: linear-gradient( to top, rgba(0, 0, 0, 0.8) 10%, transparent ), url({{ $hero['image']['data']['url'] }})"></div>
```

Comment:
The reason for the style attribute is so that the desired image (i.e $hero['image']) can be passed through as the image to use in the background image styles

Reply:
Right, thanks for the clarification, I missed that when reviewing this!